src/main/java/learnositysdk/request/Telemetry.java

```diff
@@ -97,31 +100,35 @@ private static String getSdkVersion()
 		return version;
 	}
 
-	private static String getLanguage()
+	private static String getLanguageFlavor()
 	{
 		String runtime = System.getProperty("java.runtime.name").toLowerCase(Locale.ENGLISH);
 
-		if (runtime.contains(Telemetry.RT_OPEN_JDK)) {
-			return Telemetry.RT_OPEN_JDK;
+		if (runtime.contains("openjdk")) {
+			return Telemetry.FLAV_OPEN_JDK;
+		}
+
+		if (runtime.contains("java(tm)")) {
+			return Telemetry.FLAV_ORACLE;
 		}
 
-		return Telemetry.RT_JAVA;
+		return runtime;
```

Comment:
@shtrom I've gone with returning the full value (instead of the "unknown" you proposed previously) if we encounter an unknown runtime for sake of discovery if it occurs - same as how we handle the platform.